components/automate-chef-io/Makefile

```diff
@@ -48,13 +48,13 @@ generate_swagger: $(SWAGGER_RESULT_FILE)
 		jq -es 'if . == [] then null else .[] | .paths = (.paths | with_entries( select( all( .value[].tags[]; . != "hidden") ) ) ) end' \
 		> $(SWAGGER_RESULT_FILE)
 
-serve: sync generate_swagger $(REDOC_FILE)
+serve: sync $(REDOC_FILE)
```

Comment:
Rebuilding the combined swagger file to reflect local changes is probably still helpful in the dev environment.